marlowe-dashboard-client/src/Dashboard/View.purs

```diff
@@ -128,16 +190,32 @@ renderMobileMenu menuOpen =
         iohkLinks
     ]
 
-renderFooter :: forall p. HTML p Action
-renderFooter =
+dashboardBreadcrumb :: forall p. (Maybe Contract.State) -> HTML p Action
+dashboardBreadcrumb mSelectedContractState =
+  div [ classNames [ "border-b", "border-gray" ] ]
+    [ nav [ classNames $ Css.container <> [ "flex", "gap-2", "py-2" ] ]
+        $ [ a [ onClick_ $ SelectContract Nothing ] [ text "Dashboard" ] ]
+        <> case mSelectedContractState of
+            Just { nickname } ->
+              [ span_ [ text ">" ]
+              , span_ [ text nickname ]
+              ]
+            Nothing -> []
+    ]
```

Comment:
We should probably create a function that receives an array of text and actions and draws a breadcrumb￼￼￼

Reply:
Yes, we probably should. The next PR will add a breadcrumb to some cards which will have - I think - exactly the same style. So that'll be the time to do it.